### ISSUE TITLE
Account breached?

### ISSUE THREAD

**Issue by karafecho** [NCATS-Gamma/robokop#498]
Somebody appears to be running queries under my ROBOKOP account. I did not run either of the queries below, at least not today.

 ["COP for naltrexone (CHEBI:7465) and fibromyalgia (MONDO:0005546) "]( https://robokop.renci.org/q/ba1027b8-01e8-4a07-b446-3ac8125e6768/)

[ "Capuzzi 3: COP for tolbutamide (CHEBI:27999) and Diabetes mellitus type 2 (MONDO:0005148) ".](https://robokop.renci.org/a/588fc8b5-54ec-419a-bd35-cfa72b2d5ae4_fd378cd3-4482-4032-a992-65bb863ae355/)

I may have run the naltrexone - fibromyalgia query, but I never ran a query titled "Capuzzi..." and several of those have showed up in my question library, with email associated email announcements.

**Comment by patrickkwang** [comment]
This is indeed distressing. Is it possible that you logged in with your credentials on someone else's machine? This does not look like the work of a hacker or a bot.

**Comment by karafecho** [comment]
Nope, I was on my machine, logged in under "kfecho@copperlineprofessionalsolutions.com". I suspect that the person posting the questions was in Alex's group, given the focus on Stephen Capuzzi's COP work. Not sure what happened.

**Comment by karafecho** [comment]
So, this appears to be happening again. For instance, I did not run [this query](https://robokop.renci.org/q/d7dbcfb7-e4cd-497d-9369-f4172bd0edcc/).
